ec2_security_groups_dumper/main.py

```diff
@@ -61,6 +63,7 @@ def __init__(self,
         assert isinstance(id, str), "Invalid id: {}".format(id)
         assert isinstance(name, str)
         assert isinstance(description, str)
+        assert isinstance(vpc_id, str)
```

Comment:
Taking classic into account, a firewall rule does not need to be in a VPC, so let's make this `str` or `NoneType`.